### ISSUE TITLE
RFE: filter out `<none>` images in buildah images

### ISSUE THREAD

**Issue by gouyang** [projectatomic/buildah#410]
After some testing work, it produces many `<none>` images.
How about filtering out these `<none>` images and just show these images with a name, and add `--all` to list all of them.

```
# buildah images 
IMAGE ID             IMAGE NAME                                               CREATED AT             SIZE
3f8a4339aadd         docker.io/library/nginx:latest                           Dec 26, 2017 18:17     107 MB
2ececedaea32         <none>                                                   Jan 24, 2018 02:46     228.5 MB
bb7b10df1597         <none>                                                   Jan 24, 2018 02:47     228.5 MB
5e0cf9870dd4         localhost:5000/commit/nginx:latest                       Jan 24, 2018 02:47     228.5 MB
d254289a56eb         <none>                                                   Jan 24, 2018 02:48     4.199 KB
1075f9e86d86         <none>                                                   Jan 24, 2018 02:48     4.199 KB
995f563f0c36         <none>                                                   Jan 24, 2018 02:48     4.199 KB
b02b85ac5250         <none>                                                   Jan 24, 2018 02:48     4.199 KB
3b523c8174f3         <none>                                                   Jan 24, 2018 02:48     4.199 KB
fcfba2b10cc3         <none>                                                   Jan 24, 2018 02:48     4.199 KB
558df60f373a         <none>                                                   Jan 24, 2018 02:53     228.5 MB
6098d9800339         <none>                                                   Jan 24, 2018 02:53     228.5 MB
18c2e692aefc         <none>                                                   Jan 24, 2018 02:55     4.199 KB
f9b6f7f7b9d3         docker.io/library/busybox:latest                         Jan 15, 2018 22:34     1.294 MB
f3d8230c9c2e         <none>                                                   Jan 24, 2018 03:04     228.5 MB
df090715e0d8         <none>                                                   Jan 24, 2018 03:05     228.5 MB
7e7ba81674d1         <none>                                                   Jan 24, 2018 03:06     4.199 KB
0cd5b9e990c4         <none>                                                   Jan 24, 2018 03:06     4.199 KB
381cd4db3945         <none>                                                   Jan 24, 2018 03:06     4.199 KB
9c31bb85f9e7         <none>                                                   Jan 24, 2018 03:06     4.199 KB
73d26c19dd2c         <none>                                                   Jan 24, 2018 03:06     4.199 KB
19c4675bb968         <none>                                                   Jan 24, 2018 03:06     4.199 KB
437390bbff77         <none>                                                   Jan 24, 2018 03:15     4.199 KB
4786bf0d131d         <none>                                                   Jan 24, 2018 03:15     4.199 KB
d037dbd2e576         <none>                                                   Jan 24, 2018 03:15     4.199 KB
555be1f096c2         <none>                                                   Jan 24, 2018 03:15     4.199 KB
8de0a2199731         <none>                                                   Jan 24, 2018 03:15     4.199 KB
0db630c10a10         <none>                                                   Jan 24, 2018 03:15     4.199 KB
11bab3463195         <none>                                                   Jan 24, 2018 03:22     4.199 KB
af03cdd0c1ee         <none>                                                   Jan 24, 2018 03:22     4.199 KB
335f7e7a2694         <none>                                                   Jan 24, 2018 03:22     4.199 KB
a67b6a86fbcb         <none>                                                   Jan 24, 2018 03:22     4.199 KB
5e1c35610783         <none>                                                   Jan 24, 2018 03:22     4.199 KB
```

**Comment by ripcurld0** [comment]
Basically you can use `--filter` for that job. Just type:
```
# buildah images --filter "dangling=false"
```

@TomSweeneyRedHat @rhatdan WDYT

**Comment by rhatdan** [comment]
Tat is fine or just use a pipe with grep.
We should probably add a
buildah rmi --dangling 
option

**Comment by ripcurld0** [comment]
Sounds like a good idea.

**Comment by ripcurld0** [comment]
This feature was implemented in #418.
Thanks @artmello & @gouyang
